composer/spec/dependabot/composer/update_checker_spec.rb

```diff
@@ -528,6 +528,12 @@
 
       it { is_expected.to be_nil }
 
+      it "logs an error" do
+        allow(Dependabot.logger).to receive(:error)
+        Dependabot.logger.error
```

Comment:
Isn't _this_method call fulfilling the expectation set on the line above? We should instead run the actual code under test here I think @robaiken

Reply:
Oof, good catch Jurre, I totally missed that.

Reply:
nice, thanks for spotting this